homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -48,6 +52,21 @@ async def async_setup_entry(hass, entry, async_add_entities) -> None:
 
     async_add_entities(alarms, True)
 
+    # Set up services
+    platform = entity_platform.async_get_current_platform()
+
+    platform.async_register_entity_service(
+        SERVICE_ALARM_ARM_AWAY_INSTANT,
+        None,
```

Comment:
Please pass an empty dict to have the helper generate a service schema that works with target parameter.

Reply:
Submitted https://github.com/home-assistant/core/pull/61595